includes/class-monk.php

```diff
@@ -243,6 +244,8 @@ private function define_public_hooks() {
 		$this->loader->add_action( 'pre_get_posts', $plugin_public, 'monk_public_posts_filter' );
 		$this->loader->add_filter( 'get_terms_defaults', $plugin_public, 'monk_public_terms_filter' );
 		$this->loader->add_filter( 'wp_nav_menu_args', $plugin_public, 'monk_filter_nav_menus' );
+		$this->loader->add_filter( 'pre_option_blogname', $plugin_public, 'monk_filter_options', 10, 2 );
```

Comment:
Move `monk_translatable_options` filter to here (or a better place) and loop through it adding filter for each translatable option.

Add filter documentation.